setup.py

```diff
@@ -157,7 +157,7 @@ def get_git_sha() -> str:
             "snowflake-sqlalchemy==1.2.4"
         ],  # PINNED! 1.2.5 introduced breaking changes requiring sqlalchemy>=1.4.0
         "teradata": ["sqlalchemy-teradata==0.9.0.dev0"],
-        "thumbnails": ["Pillow>=7.0.0, <8.0.0"],
+        "thumbnails": ["Pillow>=8.2.0, <9.0.0"],
```

Comment:
It appears we're already pinning to >=8.3.1 in the development deps - perhaps we could use the same version here and potentially consider bumping to 8.3.2 which addresses CVE-2021-23437?
https://github.com/apache/superset/blob/0f3b6304549c52dee6b6614446ba3ae09548f526/requirements/development.in#L21

Reply:
good point, made the change